content/docs-v1/metadata-configuration.md

```diff
@@ -17,16 +17,20 @@ You can read more about YAML syntax basics in [*YAML & Markdown*](/docs-v1/funda
 
 ## Adjust the Default Publication Settings in config.yaml File
 
-The `config.yaml` file is a required file for Quire. It is used to define a number of key values used in Quire {{< q-def "templates" >}}. Users who have worked on other non-Quire Hugo projects will note that they typically use the `config.yaml` file to also store publication metadata. Given the potentially large scope of this kind of metadata in formal digital publications, Quire instead uses the `publication.yaml` file for that purpose [(see below)](#add-and-edit-important-metadata-in-publicationyaml-file).
+The `config.yaml` file is a required file for Quire. It is used to define a number of key values, including the location of the figure image directory, format for object entry pages, bibliography format, name of navigation buttons, contributor byline styling, and more. These properties are individually documented in the [*For Developers*](/docs-v1/for-developers/).
 
-Some examples of properties outlined in the `config.yaml` include: location of the figure image directory, format for object entry pages, bibliography format, name of navigation buttons, contributor byline styling, and more. These properties are individually documented in the [*For Developers*](/docs-v1/for-developers/).
+Some users may note that the `config.yaml` file is typically where publication metadata is also stored. Given the potentially large scope of the metadata found in formal digital publications, Quire instead uses the `publication.yaml` file for that purpose ([see below](#add-and-edit-important-metadata-in-publicationyaml-file)).
```

Comment:
This was true in Hugo, but I don't think necessarily so in 11ty. I think you could delete this paragraph.